config.json

```diff
@@ -6,14 +6,6 @@
   "deprecated": [
 
   ],
-  "ignored": [
-    "_template",
-    "bin",
-    ".gradle",
-    "docs",
-    "img",
-    "build"
```

Comment:
ah, interesting. Yeah, let's figure out what's going on with that when we next see it happen.